packages/@office-iss/react-native-win32/src/Libraries/Pressability/Pressability.win32.js

```diff
@@ -286,20 +286,20 @@ const Transitions = Object.freeze({
   },
 });
 
-const isActiveSignal = signal =>
+const isActiveSignal = (signal: TouchState) =>
```

Comment:
For my education: Type annotation looks like TypeScript - should this be reflected in the file name extension?

Reply:
Nevermind, this is probably Flow.